doc/en/example/pythoncollection.rst

```diff
@@ -180,18 +180,32 @@ Finding out what is collected
 You can always peek at the collection tree without running tests like this::
 
     . $ pytest --collect-only pythoncollection.py
-    =========================== test session starts ============================
-    platform linux -- Python 3.x.y, pytest-3.x.y, py-1.x.y, pluggy-0.x.y
-    rootdir: $REGENDOC_TMPDIR, inifile: pytest.ini
-    collected 3 items
-    <Module 'CWD/pythoncollection.py'>
-      <Function 'test_function'>
-      <Class 'TestClass'>
-        <Instance '()'>
-          <Function 'test_method'>
-          <Function 'test_anothermethod'>
+    Traceback (most recent call last):
```

Comment:
Hmm this is incorrect, probably some update to regendocs?

Reply:
no idea off-hand, its the only hunk in that file, i'll revert that file and sort it out later